python/mxnet/visualization.py

```diff
@@ -59,8 +64,17 @@ def plot_network(symbol, title="plot", shape=None):
     conf = json.loads(symbol.tojson())
     nodes = conf["nodes"]
     heads = set([x[0] for x in conf["heads"]])  # TODO(xxx): check careful
+    # default attributes of node
     node_attr = {"shape": "box", "fixedsize": "true",
                  "width": "1.3", "height": "0.8034", "style": "filled"}
+    # merge the dcit provided by user and the default one
+    if len(node_attrs) is not 0:
+        for k in node_attrs.keys():
+            if k in node_attr.keys():
+                node_attr[k] = node_attrs[k]
+            else:
+                raise ValueError("Input keys should be in: \
+                shape, fixedsize, width, height, style")
```

Comment:
I guess it might be OK to just use `node_attr.update(node_attrs)` to merge whatever the user passed here, this allows the user to add extra parameters to GraphViz if needed.